packages/@webex/plugin-cc/src/features/Agent.ts

```diff
@@ -34,4 +34,19 @@ export default class Agent {
       return Promise.reject(new Error('Error while performing agent login', error));
     }
   }
+
+  public async stationLogout(options: {logoutReason: string}): Promise<LogoutSuccess> {
+    const {logoutReason} = options;
+
+    try {
+      const response = await this.agentService.stationLogout({
+        logoutReason,
```

Comment:
The options param can be passed as is to agentService without the need for destructuring

Reply:
Done